source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -2735,3 +2735,98 @@ See :ref:`minio-external-identity-management-plugin` for a tutorial on using the
       .. include:: /includes/common-minio-external-auth.rst
          :start-after: start-minio-identity-management-comment
          :end-before: end-minio-identity-management-comment
+
+
+Data Compression
+~~~~~~~~~~~~~~~~
+
+The following section documents settings for enabling data compression for objects.
+See :ref:`minio-data-compression` for tutorials on using these configuration settings.
+
+.. mc-conf:: compression
+
+   The top-level configuration key for enabling :ref:`minio-data-compression`.
+
+   Use :mc-cmd:`mc admin config set` to set or update the configuration.
+   Specify optional arguments as a whitespace (``" "``)-delimited list.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin config set compression           \
+                          [ARGUMENT=VALUE] ...  \
+
+   The following types of data are compressed by default:
+
+   +-----------------+--------------------------+
```

Comment:
Worth using the existing single-source here?

Reply:
ooo yes!